app/v3/services/TriggerBsasService.scala

```diff
@@ -29,32 +28,43 @@ import v3.models.request.triggerBsas.TriggerBsasRequest
 import v3.models.response.TriggerBsasResponse
 import v3.support.DownstreamResponseMappingSupport
 
-import scala.concurrent.{ExecutionContext, Future}
+import javax.inject.{ Inject, Singleton }
+import scala.concurrent.{ ExecutionContext, Future }
 
 @Singleton
-class TriggerBsasService @Inject()(connector: TriggerBsasConnector) extends DownstreamResponseMappingSupport with Logging{
+class TriggerBsasService @Inject()(connector: TriggerBsasConnector) extends DownstreamResponseMappingSupport with Logging {
 
-  def triggerBsas(request: TriggerBsasRequest)
-                       (implicit hc: HeaderCarrier, ec: ExecutionContext, logContext: EndpointLogContext,
-                        correlationId: String):
-  Future[Either[ErrorWrapper, ResponseWrapper[TriggerBsasResponse]]] = {
+  def triggerBsas(request: TriggerBsasRequest)(implicit hc: HeaderCarrier,
+                                               ec: ExecutionContext,
+                                               logContext: EndpointLogContext,
+                                               correlationId: String): Future[Either[ErrorWrapper, ResponseWrapper[TriggerBsasResponse]]] = {
 
     val result = for {
-      desResponseWrapper <- EitherT(connector.triggerBsas(request)).leftMap(mapDownstreamErrors(mappingDesToMtdError))
+      desResponseWrapper <- EitherT(connector.triggerBsas(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))
```

Comment:
can refactor this to: ` val result = EitherT(connector.triggerBsas(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))`